<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1537</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #001f3f;
}

QLabel {
    color: #ffffff;
    font-family: 'Segoe UI', 'Arial', sans-serif;
}

QFrame {
    background-color: #002f5f;
    border: 1px solid #00ff00;
    border-radius: 5px;
}

QListWidget {
    background-color: #001f3f;
    color: #ffffff;
    border: 1px solid #00ff00;
    font-family: 'Consolas', 'Courier New', monospace;
}

QListWidget::item {
    border-bottom: 1px solid #00ff00;
    padding: 2px;
}

QScrollBar:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    width: 15px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #00ff00;
    min-height: 20px;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    height: 15px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QMenuBar {
    background-color: #001f3f;
    color: #ffffff;
}

QMenuBar::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}

QMenu {
    background-color: #001f3f;
    color: #ffffff;
}

QMenu::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QLCDNumber" name="lcdNumber_4"/>
    </item>
    <item row="0" column="1">
     <widget class="QLCDNumber" name="lcdNumber_2"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="australiaLabel">
      <property name="text">
       <string>Australia</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="germanyLabel">
      <property name="text">
       <string>Germany</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="newyorkLabel">
      <property name="text">
       <string>New York</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLCDNumber" name="lcdNumber"/>
    </item>
    <item row="0" column="2">
     <widget class="QLCDNumber" name="lcdNumber_3"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="japanLabel">
      <property name="text">
       <string>Japan</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLCDNumber" name="lcdNumber_6"/>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="sanfransiscoLabel">
      <property name="text">
       <string>San Fransico</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="6">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1516</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="londonLabel">
      <property name="text">
       <string>London</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="startDateLabel">
      <property name="styleSheet">
       <string notr="true">QMainWindow {
    background-color: #001f3f;
}

QLabel {
    color: #ffffff;
    font-family: 'Segoe UI', 'Arial', sans-serif;
}

QFrame {
    background-color: #002f5f;
    border: 1px solid #00ff00;
    border-radius: 5px;
}

QListWidget {
    background-color: #001f3f;
    color: #ffffff;
    border: 1px solid #00ff00;
    font-family: 'Consolas', 'Courier New', monospace;
}

QListWidget::item {
    border-bottom: 1px solid #00ff00;
    padding: 2px;
}

QScrollBar:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    width: 15px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #00ff00;
    min-height: 20px;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    height: 15px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QMenuBar {
    background-color: #001f3f;
    color: #ffffff;
}

QMenuBar::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}

QMenu {
    background-color: #001f3f;
    color: #ffffff;
}

QMenu::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}
</string>
      </property>
      <property name="text">
       <string>Start Date</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLCDNumber" name="lcdNumber_5"/>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QDateEdit" name="dateEdit">
      <property name="styleSheet">
       <string notr="true">QMainWindow {
    background-color: #001f3f;
}

QLabel {
    color: #ffffff;
    font-family: 'Segoe UI', 'Arial', sans-serif;
}

QFrame {
    background-color: #002f5f;
    border: 1px solid #00ff00;
    border-radius: 5px;
}

QListWidget {
    background-color: #001f3f;
    color: #ffffff;
    border: 1px solid #00ff00;
    font-family: 'Consolas', 'Courier New', monospace;
}

QListWidget::item {
    border-bottom: 1px solid #00ff00;
    padding: 2px;
}

QScrollBar:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    width: 15px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #00ff00;
    min-height: 20px;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    height: 15px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QMenuBar {
    background-color: #001f3f;
    color: #ffffff;
}

QMenuBar::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}

QMenu {
    background-color: #001f3f;
    color: #ffffff;
}

QMenu::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="6">
     <widget class="QColumnView" name="metricsView">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="2">
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="styleSheet">
       <string notr="true">QMainWindow {
    background-color: #001f3f;
}

QLabel {
    color: #ffffff;
    font-family: 'Segoe UI', 'Arial', sans-serif;
}

QFrame {
    background-color: #002f5f;
    border: 1px solid #00ff00;
    border-radius: 5px;
}

QListWidget {
    background-color: #001f3f;
    color: #ffffff;
    border: 1px solid #00ff00;
    font-family: 'Consolas', 'Courier New', monospace;
}

QListWidget::item {
    border-bottom: 1px solid #00ff00;
    padding: 2px;
}

QScrollBar:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    width: 15px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #00ff00;
    min-height: 20px;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    height: 15px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QMenuBar {
    background-color: #001f3f;
    color: #ffffff;
}

QMenuBar::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}

QMenu {
    background-color: #001f3f;
    color: #ffffff;
}

QMenu::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="sectorLabel">
      <property name="styleSheet">
       <string notr="true">QMainWindow {
    background-color: #001f3f;
}

QLabel {
    color: #ffffff;
    font-family: 'Segoe UI', 'Arial', sans-serif;
}

QFrame {
    background-color: #002f5f;
    border: 1px solid #00ff00;
    border-radius: 5px;
}

QListWidget {
    background-color: #001f3f;
    color: #ffffff;
    border: 1px solid #00ff00;
    font-family: 'Consolas', 'Courier New', monospace;
}

QListWidget::item {
    border-bottom: 1px solid #00ff00;
    padding: 2px;
}

QScrollBar:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    width: 15px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #00ff00;
    min-height: 20px;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    height: 15px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QMenuBar {
    background-color: #001f3f;
    color: #ffffff;
}

QMenuBar::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}

QMenu {
    background-color: #001f3f;
    color: #ffffff;
}

QMenu::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}
</string>
      </property>
      <property name="text">
       <string>Sector</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QComboBox" name="sectorComboBox">
      <property name="styleSheet">
       <string notr="true">QMainWindow {
    background-color: #001f3f;
}

QLabel {
    color: #ffffff;
    font-family: 'Segoe UI', 'Arial', sans-serif;
}

QFrame {
    background-color: #002f5f;
    border: 1px solid #00ff00;
    border-radius: 5px;
}

QListWidget {
    background-color: #001f3f;
    color: #ffffff;
    border: 1px solid #00ff00;
    font-family: 'Consolas', 'Courier New', monospace;
}

QListWidget::item {
    border-bottom: 1px solid #00ff00;
    padding: 2px;
}

QScrollBar:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    width: 15px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #00ff00;
    min-height: 20px;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    height: 15px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QMenuBar {
    background-color: #001f3f;
    color: #ffffff;
}

QMenuBar::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}

QMenu {
    background-color: #001f3f;
    color: #ffffff;
}

QMenu::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}
</string>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="6">
     <widget class="QColumnView" name="tradesView"/>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="endDateLabel">
      <property name="styleSheet">
       <string notr="true">QMainWindow {
    background-color: #001f3f;
}

QLabel {
    color: #ffffff;
    font-family: 'Segoe UI', 'Arial', sans-serif;
}

QFrame {
    background-color: #002f5f;
    border: 1px solid #00ff00;
    border-radius: 5px;
}

QListWidget {
    background-color: #001f3f;
    color: #ffffff;
    border: 1px solid #00ff00;
    font-family: 'Consolas', 'Courier New', monospace;
}

QListWidget::item {
    border-bottom: 1px solid #00ff00;
    padding: 2px;
}

QScrollBar:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    width: 15px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #00ff00;
    min-height: 20px;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: 1px solid #00ff00;
    background: #001f3f;
    height: 15px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QMenuBar {
    background-color: #001f3f;
    color: #ffffff;
}

QMenuBar::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}

QMenu {
    background-color: #001f3f;
    color: #ffffff;
}

QMenu::item:selected {
    background-color: #00ff00;
    color: #001f3f;
}
</string>
      </property>
      <property name="text">
       <string>End Date</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="6">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1516</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" colspan="6">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="overviewTab">
       <attribute name="title">
        <string>Overview</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="metricsView">
       <attribute name="title">
        <string>Metrics</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="chart1View">
       <attribute name="title">
        <string>Chart1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="chart2Tab">
       <attribute name="title">
        <string>Chart2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="marketLabel">
      <property name="text">
       <string>Market</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="2">
     <widget class="QComboBox" name="marketComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1537</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUpdate"/>
    <addaction name="actionDelete_File"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionDelete_File">
   <property name="text">
    <string>Delete File</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
